satellite/satellitedb/metainfo.sql

```diff
@@ -0,0 +1,116 @@
+CREATE TYPE encryption_parameters AS (

```

Comment:
Is this supported by sqlite or we will drop support for satellite?

Reply:
This will be adjusted as necessary.